Esri/arcgis-maps-sdk-swift-samples
# [New] Apply mosaic rule to rasters

## Description

This PR implements **`Apply mosaic rule to rasters`** in `Layers` category.

`swift/issues/5481`

## How To Test

Load map and wait for raster images to load. Tap rules in the navbar and select a rule. The raster overlaying the map should update to reflect the rule. 

## Screenshots
![apply-mosaic-rule-to-rasters](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/edec47a2-18c6-4a51-8c31-ca1cb6cc0598)
